aptos-move/framework/aptos-stdlib/sources/data_structures/smart_vector.spec.move

```diff
@@ -22,19 +26,38 @@ spec aptos_std::smart_vector {
 
     spec destroy_empty {
         aborts_if !(is_empty(v));
-        aborts_if vector::length(v.inline_vec) != 0
+        aborts_if len(v.inline_vec) != 0
             || option::is_some(v.big_vec);
     }
 
     spec borrow {
         aborts_if i >= length(v);
         aborts_if option::is_some(v.big_vec) && (
-            (vector::length(v.inline_vec) + big_vector::length<T>(option::borrow(v.big_vec))) > MAX_U64
+            (len(v.inline_vec) + big_vector::length<T>(option::borrow(v.big_vec))) > MAX_U64
         );
     }
 
     spec push_back<T: store>(v: &mut SmartVector<T>, val: T) {
+        // use aptos_std::big_vector;
+        // use aptos_std::type_info;
         pragma verify = false; // TODO: set to false because of timeout
+        // pragma aborts_if_is_partial;
```

Comment:
Comment them out because of timeout

Reply:
This function already has `pragma verify = false;`. Did `pragma aborts_if_is_partial;` still affect it?